src/AnnotationThread.js

```diff
@@ -616,6 +618,17 @@ class AnnotationThread extends EventEmitter {
         this.deleteAnnotation(data.annotationID);
     }
 
+    setThreadBoundary() {
```

Comment:
Docs. Also, could this be named `updateThreadBoundary` or something else? When I see `set`, I want to pass in some parameters to it.